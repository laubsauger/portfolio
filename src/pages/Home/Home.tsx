import React from 'react';

function Home() {
  return (
    <div data-testid="home-page">
      Home
    </div>
  );
}

export default Home;
