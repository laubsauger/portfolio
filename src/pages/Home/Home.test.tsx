import * as React from 'react';
import Home from './Home';
import {render} from "@testing-library/react";

describe('Home', () => {
  it('renders', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});