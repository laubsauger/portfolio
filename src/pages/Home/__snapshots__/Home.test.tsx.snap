// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<DocumentFragment>
  <div
    data-testid="home-page"
  >
    Home
  </div>
</DocumentFragment>
`;
